resources/lib/apihelper.py

```diff
@@ -601,6 +601,14 @@ def get_episodes(self, program=None, season=None, episodes=None, category=None,
             facets = search_json.get('facets', {}).get('facets')
             if facets:
                 seasons = next((f.get('buckets', []) for f in facets if f.get('name') == 'seasons' and len(f.get('buckets', [])) > 1), None)
+            # Experimental: VRT Search API only returns a maximum of 10 seasons, to get all seasons we need to use the "model.json" API
+            if seasons and program and len(seasons) == 10:
+                seasons = []
+                season_json = get_url_json('https://www.vrt.be/vrtnu/a-z/%s.model.json' % program)
+                season_items = season_json.get(':items').get('parsys').get(':items').get('container') \
+                                          .get(':items').get('banner').get(':items').get('navigation').get(':items')
```

Comment:
My concern is that this can easily break. It feels as if the data structure was modelled for a very specific use in the front-end, and for that reason not a good API to be used for anything structural.

In this case I am not so much concerned with that it stops working as it is a very marginal issue and a workaround, but more about how it will show an error that the user may not understand or feel the add-on is not reliable. So I would prefer either that it continues as before without throwing an error, or gives a pop-up that is clear the user should report it.

Reply:
Thanks, I added a try...except block to catch the AtrributeError.
When the json would change, it continues as before.